### DISCUSSION TITLE
Proxy stream without path

### DISCUSSION THREAD

**Discussion by bonuzzz** [bluenviron/mediamtx#977] [Questions]
I want to proxy stream, but without having path on the destination link, only ip:port. Is it possible to do that and how?

**Comment by aler9** [comment]
Some cameras allow to read their streams without using paths; you can read a camera stream without using a path, i. e.

```
rtsp://camera-url:camera-port
```

You can use that URL in the `source` parameter:

```yml
paths:
  mycamera:
    source: rtsp://camera-url:camera-port
```

Regarding the server, it works with paths, therefore you have to use a path in order to read a proxied stream, i.e.

```
rtsp://server-url:8554/mycamera
```

If you don't like paths at all you can [build a custom server with gortsplib](https://github.com/aler9/gortsplib/tree/main/examples/server).

[a comment by an automated account is not shown]
